components/create/MintConfirmModal.vue

```diff
@@ -0,0 +1,265 @@
+<template>
+  <NeoModal
+    :value="value"
+    :can-cancel="['outside', 'escape']"
+    scroll="clip"
+    class="top"
+    max-height="calc(100vh - 180px)"
+    @close="onClose">
+    <div class="modal-width">
+      <header
+        class="modal-card-head is-flex is-justify-content-space-between is-align-items-center border-bottom">
+        <span class="modal-card-title is-size-6 has-text-weight-bold">
+          {{ $t('mint.nft.modal.action') }}
+        </span>
+        <NeoButton
+          class="py-1 px-2"
+          variant="text"
+          no-shadow
+          icon="xmark"
+          size="medium"
+          @click.native="onClose" />
+      </header>
+      <div v-if="isLogIn" class="px-6 pt-4">
+        <div
+          class="rounded border shade-border-color is-flex is-justify-content-start is-flex-grow-1 pl-3">
+          <IdentityItem
+            :label="$t('confirmPurchase.connectedWith')"
+            hide-identity-popover
+            disable-identity-link
+            :prefix="urlPrefix"
+            :account="accountId"
+            class="identity-name-font-weight-regular"
+            data-testid="item-creator" />
+        </div>
+      </div>
+      <div class="px-6 mt-4 has-text-weight-bold">
+        {{ title }}
+      </div>
+      <div class="py-4">
+        <ConfirmMintItem :nft="extendedInformation" class="px-6" />
+      </div>
+      <div class="px-6">
+        <PriceItem :nft="extendedInformation" />
+      </div>
```

Comment:
can we fix types here, on `:nft="extendedInformation"` prop

`Type '{ networkFee: number; existentialDeposit: number; kodadotFee: number; kodadotUSDFee: number; carbonlessFee: number; carbonlessUSDFee: number; totalFee: number; totalUSDFee: number; blockchain: Option | undefined; ... 6 more ...; paidToken: ChainProperties; }' is not assignable to type 'ExtendedInformation'.ts(2322)`